src/resources/GlobalGroups/GlobalGroupInterfaces.ts

```diff
@@ -12,7 +12,10 @@ export interface GlobalGroupModel {
 }
 
 export interface GlobalPrivilegeModel extends PrivilegeModel {
-    level: string;
```

Comment:
Level is not mandatory on all Privilege owner, targetDomain, and type [Combinations](https://docs.coveo.com/en/159/)